### PR TITLE
docs(google-maps/map-advanced-marker): README.md added

### PR DESCRIPTION
**Pull request by K0n4ta13**
I add the README file.

### CODE REVIEW

File: src/google-maps/map-advanced-marker/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# MapAdvancedMarker
+
+The `MapAdvancedMarker` component wraps the [`google.maps.marker.AdvancedMarkerElement` class](https://developers.google.com/maps/documentation/javascript/reference/advanced-markers) from the Google Maps JavaScript API. The `MapAdvancedMarker` component displays a marker on the map when it is a content child of a `GoogleMap` component. Like `GoogleMap`, this component offers an `options` input as well as convenience inputs for `title`, `position`, `content`, `gmpDraggable` and `zIndex`, and supports all `google.maps.marker.AdvancedMarkerElement` events as outputs.
```
Comment (crisbeto) [inline src/google-maps/map-advanced-marker/README.md:3]: I think we can drop the last sentence here. We shouldn't list out the individual inputs/outputs.

File: src/google-maps/map-advanced-marker/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+**Note:** Use of `map-advanced-marker` requires a `google-map` whth a valid `mapId`.
+
```
Comment (crisbeto) [inline src/google-maps/map-advanced-marker/README.md:6]: whth -> with

File: src/google-maps/map-advanced-marker/README.md (hunk 1 above)
Comment (K0n4ta13) [reply src/google-maps/map-advanced-marker/README.md]: Last sentence deleted.

Comment (K0n4ta13) [reply src/google-maps/map-advanced-marker/README.md]: Apologies for splitting this into two commits. I accidentally left the outputs and the options part in the first one.

**Review by crisbeto — APPROVED** [review]

**Pull request merged**